components/Layout/Layout.jsx

```diff
@@ -1,16 +1,22 @@
 import Head from "next/head";
-import { Container, Flex } from "@chakra-ui/react";
+import { Container, Flex, Text } from "@chakra-ui/react";
 import { productionUrl } from "../../utils/constants";
 
+const titleBarHeight = "80px";
+
 export default function Layout({ children }) {
+
   return (
     <>
       <Head>
         <title>Emissions Calculator</title>
         <meta name="description" content="Emissions calculator" />
         <link rel="icon" href={`${productionUrl}/favicon.ico`} />
       </Head>
-      <Flex height="100vh" alignItems="center">
+      <Flex alignItems="center" minHeight={`calc(100vh - ${titleBarHeight})`} pos="relative" top={titleBarHeight} py={5}>
```

Comment:
I'm a bit crook, so my brain might not be working so well, but I'm wondering why the container flex has that minHeight calc? The thing looks good, so I get the feeling you are getting this right, but for some reason it's not quite sitting right with me.

Reply:
I find it awkward as well. 
The need for it stems from making the header fixed, so it'd be overlapping the content container if I hadn't offset the container to the header's height. This offset means the minimum height of the contenthas to be `100vh - header.height` to ensure the header+content fill at least the entire height of the viewport.
I couldn't think of a less awkward way to accomplish this, but I wouldn't be surprised if I'm over-complicating the base solution in the first place 😅